tools/server-spring/src/main/java/org/eclipse/rdf4j/http/server/ServerInterceptor.java

```diff
@@ -24,13 +28,23 @@
  */
 public abstract class ServerInterceptor implements HandlerInterceptor {
 
+	private static final String REQUEST_ID_KEY = "requestId";
```

Comment:
I'm having some second doubts about the naming. We have users that extend the RDF4J server code and it could be that they are already doing something similar to this. Do you think we could call ours `rdf4jRequestId` just to be sure that we don't overwrite someone else's request ID?

Reply:
Ahh fair point. I've also seen the package name convention used. Something like `org.eclipse.rdf4j.requestId` so there would never be the possibility of a clash unless we did it to ourselves?

Reply:
Package name looks very nice!